integrations/datadog/client.py

```diff
@@ -172,6 +173,26 @@ async def get_slos(self) -> AsyncGenerator[list[dict[str, Any]], None]:
             yield slos
             offset += limit
 
+    async def list_slo_histories(
+        self, from_ts: int, to_ts: int
+    ) -> AsyncGenerator[list[dict[str, Any]], None]:
+        histories = []
+        async for slos in self.get_slos():
+            histories = await asyncio.gather(
+                *(self.get_slo_history(slo["id"], from_ts, to_ts) for slo in slos)
+            )
+            yield histories
```

Comment:
will it help to stream this requests?

Reply:
can you fetch multiple get_slo_history together in the same request?

Reply:
stream won't help, we want to return in batches